dspace/src/main/config/build.xml

```diff
@@ -748,7 +748,7 @@ Common usage:
 
         <mkdir dir="${handle.dir}" />
 
-        <mkdir dir="${log.dir}" />
+        <mkdir dir="${dspace.dir}/log" />
```

Comment:
While I understand your point, this would be hard to support in the DSpace install process. If someone changes the "log.dir" property (in log4j.properties) to a location outside of the `[dspace]` installation location, then there would be no guarrantee that the user running Ant would have permissions to create that "log.dir" directory.

So, essentially, I understand your idea. But, I don't think we should read the "log.dir" from log4j.properties, as there's no way to guarantee we can actually create that directory in a different location (which could cause the Ant build process to throw errors).